mrmap/quality/tables.py

```diff
@@ -21,5 +29,50 @@ class Meta:
         fields = ('metadata', 'config', 'passed', 'report', 'created_at', 'created_by_user', 'owned_by_org')
         prefix = 'conformity_check_run-table'
 
-    def render_report(self, record):
+    @staticmethod
+    def render_report(record):
         return Link(url=record.get_report_url(), content='Validation report').render(safe=True)
+
+
+class ConformityCheckRunExtraFieldsTable(tables.Table):
+    latest_check = tables.Column(verbose_name=_('Latest check'),
+                                 attrs={"th": {"class": "col-sm-1"}},
+                                 empty_values=[])
+
+    total_check_count = tables.Column(verbose_name=_('Total check count'),
+                                      attrs={"th": {"class": "col-sm-1"}},
+                                      empty_values=[])
+
+    class Meta:
+        model = ConformityCheckRun
+        fields = ["latest_check", "total_check_count"]
+        prefix = "conformity_check_run-extra-fields-table"
+
+    @staticmethod
+    def render_latest_check(record):
+        latest_check = ConformityCheckRun.objects.get_latest_check(record)
```

Comment:
This will cause for every row one extra sql call. You can do it better by optimizing the queryset for this table, so the last check is included by an annotation. 

Example:
```python
class ConformityCheckRunManager(models.Manager):

    def get_latest_check(self):
         return self.get_queryset().filter(metadata__pk=OuterRef('pk')).latest()  # do not call latest with params... use class meta option get_latest_by

    def for_table_view(self):
            return super().get_queryset().annotate(latest_check=self.get_latest_check())  



class ConformityCheckRunListView(SecuredListMixin, FilterView):
    model = ConformityCheckRun
    table_class = ConformityCheckRunTable
    queryset = ConformityCheckRun.objects.for_table_view()
```

after that you can use the value instead of the record in the rendering function of the table:

```python
def render_latest_check(value):
    # value is the latest_check attribute from the annotated queryset.
   ...
```